modules/test-utils/src/browser-automation/browser-test-driver.ts

```diff
@@ -131,16 +135,6 @@ export default class BrowserTestDriver extends BrowserDriver {
     );
   }
 
-  _startServer(config: BrowserTestDriverProps | Function): Promise<string | null> {
-    if (!config) {
```

Comment:
@Pessimistress 

- Hard to see why we need config to be a function. If we do, we might as well support it in the base class rather than overriding startServer here.
- If we need to handle empty configs, then we should have a comment explaining why